### PR TITLE
fix blend intensity

### PR DESCRIPTION
**Pull request by kmilo9999**
remove intensity calculation based on maximum value per group of voxels. changed to individual value per voxel

### CODE REVIEW

**Review by RobertGemmaJr — CHANGES_REQUESTED** [review]
Looks great! I would lint the tabbing to be safe.

Make sure you do a version bump in `package.json`, otherwise the new release won't work.

File: package.json [hunk 1]
```diff
@@ -29,9 +29,14 @@
     "@reach/listbox": "^0.16.2",
     "@reach/tabs": "^0.16.4",
     "aframe": "^1.2.0",
+    "bootstrap": "^4.6",
```
Comment (RobertGemmaJr) [inline package.json:32]: Only the version bump should have changed, not sure what's going on with all the dependencies
```suggestion
```

File: package.json [hunk 2]
```diff
@@ -29,9 +29,14 @@
...
     "d3-scale": "^4.0.2",
     "lodash": "^4.17.21",
-    "rc-slider": "^10.0.0"
+    "rc-slider": "^10.0.0",
+    "react-bootstrap": "^1.6.3",
+    "sass": "^1.44.0",
+    "styled-components": "^5.0.0",
+    "web-vitals": "^1.0.1"
```
Comment (RobertGemmaJr) [inline package.json:39]:
```suggestion
```

File: package.json [hunk 3]
```diff
@@ -52,6 +57,7 @@
     "gh-pages": "^3.2.3",
     "npm-run-all": "^4.1.5",
     "prettier": "^2.4.1",
+    "prettier-plugin-glsl": "^0.0.4",
```
Comment (RobertGemmaJr) [inline package.json:60]: Can you change this to a dev dependency?

```console
npm uninstall prettier-plugin-glsl
```

```console
npm install prettier-plugin-glsl --save-dev

File: src/aframe/fragment-shader.frag [hunk 4]
```diff
@@ -99,81 +99,74 @@ void main() {
...
-
+        if(vFragColor.a>=.95)break;
+        
         // Advance point
-        data_position += ray_direction * step_size;
+        data_position+=ray_direction*step_size;
     }
-    fragColor = vFragColor;
+    fragColor=vFragColor;
 }
```
Comment (RobertGemmaJr) [inline src/aframe/fragment-shader.frag:172]: Just adding a newline to the end of the file
```suggestion
}

```

**Review by RobertGemmaJr — COMMENTED** [review]
Looks great! Something messed up with the dependencies on your local repo though. I added the suggested changes to fix them and then it's good to go!

File: package.json (hunk 1 above)
Comment (kmilo9999) [reply package.json]: I removed it

File: package.json (hunk 2 above)
Comment (kmilo9999) [reply package.json]: I removed them

File: src/aframe/fragment-shader.frag (hunk 4 above)
Comment (kmilo9999) [reply src/aframe/fragment-shader.frag]: I added the new line at the end of the file. The spaces between the equals sign were already there.

**Review by RobertGemmaJr — APPROVED** [review]
Perfect! I'll merge and create the new release

**Pull request merged**